modvendor/modvendor_report.go

```diff
@@ -64,12 +65,15 @@ func (r Report) String() string {
 		report = append(report, fmt.Sprintf(msg, args...))
 	}
 
-	sources := []string{}
+	sources := []project.Path{}
```

Comment:
We have the project.Paths type, maybe it could also provide sorting facilities and be used here ?